gitpod-io/template-ocaml
# Add development tooling and hello world example

## Description

This extends the OCaml template to install some developer tooling related packages (ocaml-lsp, ocamlformat, ocamlformat-rpc) as well as a small hello world example. 

The selected packages and the example code is from the official [Up and Running with OCaml](https://ocaml.org/learn/tutorials/up_and_running.html) documentation.

## Related Issue(s)

None

## How to test


Start a new workspace off this branch. Open helloworld.ml and write some code to ensure that code-completion, compile errors, etc. all works as expected. Use dune to build and run the helloworld program as described in the README.

## Release Notes

```release-note
NONE
```

## Documentation
